tika-core/src/main/java/org/apache/tika/metadata/filter/CompositeMetadataFilter.java

```diff
@@ -18,7 +18,7 @@
 
 import org.apache.tika.exception.TikaException;
 import org.apache.tika.metadata.Metadata;
-
```

Comment:
Maybe Tika is using separating import groups. We should keep (and add) empty lines between those groups